Makefile

```diff
@@ -90,7 +90,7 @@ post-install:
 start: | pre-run ## Starts the React Native packager server
 	@if [ $(shell ps -e | grep -i "cli.js start" | grep -civ grep) -eq 0 ]; then \
 		echo Starting React Native packager server; \
-		node ./node_modules/react-native/local-cli/cli.js start; \
+		node ./node_modules/react-native/local-cli/cli.js start --config ../../../../packager/config.js; \
```

Comment:
I forget if I mentioned this before, but it might be nice to have an alias for this so we only have to add this flag in one place instead of everywhere we call it.